packages/protoplugin-example/src/protoc-gen-twirp-es.ts

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env -S npx tsx
+
+import { createEcmaScriptPlugin, runNodeJs } from "@bufbuild/protoplugin";
+import { version } from "../package.json";
+import {
+  literalString,
+  makeJsDoc,
+  localName,
+} from "@bufbuild/protoplugin/ecmascript";
+import { MethodKind } from "@bufbuild/protobuf";
+import type { Schema } from "@bufbuild/protoplugin/ecmascript";
+
+const protocGenTwirpEs = createEcmaScriptPlugin({
+  name: "protoc-gen-twirp-es",
+  version: `v${String(version)}`,
+  generateTs,
+});
+
+// prettier-ignore
+function generateTs(schema: Schema) {
+  for (const file of schema.files) {
+    const f = schema.generateFile(file.name + "_twirp.ts");
+    f.preamble(file);
+    const {
+      Message,
+      JsonValue
+    } = schema.runtime;
+    // Convert the Message ImportSymbol to a type-only ImportSymbol
+    const MessageAsType = Message.toTypeOnly();
```

Comment:
@timostamm Had to convert this `toTypeOnly` manually, but I didn't have to do that for `JsonValue`.  Is this expected or is this a bug?

Reply:
Yes, this is expected, we can't do better here. `JsonValue` is a type definition and can only be used as a type, so we already set it to type-only in `schema.runtim.JsonValue`. `Message` on the other hand can be used as both.